charts/firefly/templates/core/prometheusrule.yaml

```diff
@@ -0,0 +1,38 @@
+{{/*
+  Copyright © 2022 Kaleido, Inc.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://swww.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+{{- if and .Values.core.metrics.prometheusRule.enabled .Values.config.metricsEnabled .Values.core.metrics.serviceMonitor.enabled}}
+{{- if not (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
+# WARNING: prometheus-operator is not installed but prometheusRule has been enabled, this will fail. Please install
+#          prometheus-operator to resolve this.
+{{- end }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "firefly.fullname" . }}
+  labels:
+  {{- include "firefly.coreLabels" . | nindent 4 }}
+spec:
+  groups:
+    - name: {{ include "firefly.fullname" . }}
+      rules: |
+        {{- include "firefly.defaultPrometheusRules" . | nindent 4 }}
+        {{- if .Values.core.metrics.prometheusRule.rules }}
+        {{- tpl .Values.core.metrics.prometheusRule.rules . | nindent 4 }}
+        {{- end }}
```

Comment:
```suggestion
      rules:
        {{- tpl .Values.core.metrics.prometheusRule.rules . | nindent 4 }}
```

Reply:
Note the `rules` needs to be YAML not a multi-line string, think we just got it criss crossed between here and the values.